Sources/Spine/Animation.swift

```diff
@@ -249,7 +249,7 @@ extension Animation {
                     return
                 }
                 
-                let delta = timingFunction(Float(time))
+                let delta = timingFunction(Float(time) / Float(duration))
```

Comment:
We have to check `duration` if it zero or not. (Because this data is from json and can be zero, but we can't divide by zero)

Something like this:
```swift 
let deltaTime = duration > 0 ? Float(time) / Float(duration) : 0
let delta = timingFunction(deltaTime)
```